components/Tabs/src/MDCTabBarDisplayDelegate.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "MDCTabBar.h"
+
+/**
+ An additional delegate protocol for MDCTabBar that provides information about when UITabBarItems
+ are about to be displayed and when they stop being displayed.
+ */
+@protocol MDCTabBarDisplayDelegate <NSObject>
+
+/**
+ Called before a UITabBarItem is displayed.
```

Comment:
Let's make this a bit more ambiguous in the wording here. Consider adding something like this: this method is called sometime before the tab's view is displayed.  

I believe at the moment it's called when the view is loaded and configured for display, but not necessarily when it's on-screen.